frontend/components/CompactNavigationBar.js

```diff
@@ -68,21 +83,59 @@ const CompactNavigationBar = () => {
   };
 
   const menuItemClick = () => {
-    setAboutUsMenuOpen(false);
-    setMediaMenuOpen(false);
+    setAboutUsDropdownOpen(false);
+    setMediaDropdownOpen(false);
     setMenuOpen(false);
```

Comment:
For the menu item icon (e.g. the three lines and the cross) specifically, can we make these icons white? Red is reserved for links